src/game/common_defines.h

```diff
@@ -89,6 +89,8 @@ enum
     PILOT_RAVEN,
     PILOT_KREISSACK,
     NUMBER_OF_PILOT_TYPES,
+    // Major Kreissack is not a playable character
+    NUMBER_OF_PLAYABLE_PILOT_TYPES = NUMBER_OF_PILOT_TYPES - 1
```

Comment:
Best to end all enums with a comma, in case there will be new entries later, there will be less noise in the diff.